src/test/software/amazon/event/ruler/Benchmarks.java

```diff
@@ -746,6 +775,12 @@ public void CL2Benchmark() throws Exception {
 
         bm = new Benchmarker();
 
+        bm.addRules(ANYTHING_BUT_WILDCARD_RULES, ANYTHING_BUT_WILDCARD_MATCHES);
+        bm.run(citylots2);
+        System.out.println("ANYTHING-BUT-WILDCARD events/sec: " + String.format("%.1f", bm.getEPS()));
```

Comment:
anything worth updating on the readme ?

Reply:
Nah cause README mentions top-level anything-but, but doesn't go into the subtypes. Which I think is fine, because they all seem to have the same performance.